src/reducers/content.js

```diff
@@ -9,6 +9,11 @@ export default (state = initialState, action) => {
     case CONTENT_LOADED: {
       return {
         ...state,
+        includes: action.payload.contentList.included.reduce((acc, cur) => {
```

Comment:
Maybe we could add a comment to describe what we are doing and/or rename the variables. I had to actually go with debugger and look what we are doing here to understand this.

Reply:
+1 for renaming "cur" variables. How about using "node" here?